prebuilt-tasks/src/test/java/com/redhat/parodos/tasks/git/GitCloneTest.java

```diff
@@ -0,0 +1,112 @@
+package com.redhat.parodos.tasks.git;
+
+import com.redhat.parodos.workflows.work.WorkContext;
+import com.redhat.parodos.workflows.work.WorkStatus;
+import junit.framework.TestCase;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.lib.Repository;
+import org.eclipse.jgit.storage.file.FileRepositoryBuilder;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+public class GitCloneTest extends TestCase {
```

Comment:
same question about `TestCase`

Reply:
To be able to have assertTrue, AsserEquals in the class, is how idea generates it